CHANGELOG.md

```diff
@@ -1,3 +1,44 @@
+# 1.0.10-rc1 (2020-06-25)
```

Comment:
Should we add start adding links to the release page in this changelog? Exactly like we do for our links from iotedge/release page to azure-iotedge/release page:

_Head to the [product release page](https://github.com/Azure/azure-iotedge/releases) for the changelog and IoT Edge daemon binaries._

Reply:
It would just be repeatedly copied for each release.

Reply:
We can find a way to pin it to the top or something.